rubin_sim/maf/runComparison/runComparison.py

```diff
@@ -233,6 +233,9 @@ def addSummaryStats(self, metricDict=None, verbose=False):
             else:
                 self.summaryStats = self.summaryStats.join(tempStats, how='outer', lsuffix='_x')
 
+        self.summaryStats.index.name = 'run_name'
+        self.summaryStats.columns.name = 'metric'
```

Comment:
I am a big fan of assigning names to indexes and columns in pandas. They are not only useful for documentation, they make other operations a lot more convenient: it makes access to individual levels of a multi-level indexes and columns a lot more intuitive and less error-prone, and it means that columns created using reset_index() actually have meaningful names.

Flipping which "columns" are the index (or levels of the index) and which are actual columns using set_index() and reset_index() can be very useful: slicing and joining on the index is not only syntactically nicer than querying on columns, but (in some circumstances) can be much, much faster, because pandas uses fancy indexing tricks developed for databases if the index is used, but has to fall back on numpy array operations (which have to look at every row) if queries on columns are used.